pom.xml

```diff
@@ -55,9 +55,9 @@
     <jaxb.version>2.1.9</jaxb.version>
 
     <!-- Spring (DI / Rest / Security) -->
-    <spring-boot.version>1.4.0.RELEASE</spring-boot.version>
-    <spring.version>4.3.2.RELEASE</spring.version>
-    <spring.security.version>3.2.5.RELEASE</spring.security.version>
+    <spring-boot.version>1.5.21.RELEASE</spring-boot.version>
```

Comment:
If you have tested that with this update here everything continues to work, then this PR is ok for me.

Reply:
I haven't encountered problems with 1.5 so far. It seems to be backward compatible with 1.4 on almost everything.